goodtables/pipeline/batch.py

```diff
@@ -26,7 +28,7 @@ class Batch(object):
     """
 
     def __init__(self, source, source_type='csv', data_key='data',
-                 schema_key='schema', pipeline_options=None,
+                 schema_key='schema', sleep=1, pipeline_options=None,
```

Comment:
I don't want a default of 1 second - that can really slow things down for people working on local files. Please make the default `None`, so the logic is to only sleep when this is a number.